lib/AST/Module.cpp

```diff
@@ -757,8 +757,10 @@ void ModuleDecl::updateSourceFileLocationMap() {
   }
 
   // Next, add all of the macro expansion files.
-  for (auto *sourceFile : AuxiliaryFiles)
-    sourceFileLocationMap->allSourceFiles.push_back(sourceFile);
+  for (auto *sourceFile : AuxiliaryFiles) {
+    if (sourceFile->getBufferID())
```

Comment:
This is the only use of `AuxiliaryFiles` and in general it doesn't really make sense to me to have an auxiliary file without a backing buffer. IMO we should just not add the auxiliary file in `getClangSwiftAttrSourceFile`.